test/pkg.jl

```diff
@@ -41,7 +41,6 @@ temp_pkg_dir() do
   Pkg.rm("Example")
   @test isempty(Pkg.installed())
   @test !isempty(Pkg.available("Example"))
-  @test_throws ErrorException Pkg.available("FakePackageDoesn'tExist")
```

Comment:
Many `Pkg` calls are executed on a collection packages, so if something happened with the package the execution would continue on the rest. But, you are right. It was silly of me to remove the test because it can be addressed to submodule call, `Pkg.Entry.available`. I'll correct it.